CHANGES.md

```diff
@@ -0,0 +1,6303 @@
+    loop ([#2449](https://github.com/gap-system/gap/pull/2449)).
+
+  - Fixed a crash when an error occurs and `OutputLogTo` (Reference:
+    OutputLogTo for streams) points to a stream which internally uses
+    another stream
+    ([#2596](https://github.com/gap-system/gap/pull/2596)).
+
+Fixed bugs that could lead to incorrect results:
+
+  - Fixed a bug in computing maximal subgroups, which broke some other
+    calculations, in particular, computing intermediate subgroups.
+    ([#2488](https://github.com/gap-system/gap/pull/2488)). \[Reported
+    by Seyed Hassan Alavi\]
+
+Other fixed bugs and further improvements:
+
+  - Profiling now correctly handles calls to `longjmp` and allows to
+    generate profiles using version 2.0.1 of the **Profiling** package
+    ([#2444](https://github.com/gap-system/gap/pull/2444)).
+
+  - The `bin/gap.sh` script now respects the `GAP_DIR` environment
+    variable ([#2465](https://github.com/gap-system/gap/pull/2465)).
+    \[Contributed by RussWoodroofe\]
+
+  - The `bin/BuildPackages.sh` script now properly builds binaries for
+    the **simpcomp** package
+    ([#2475](https://github.com/gap-system/gap/pull/2475)).
+
+  - Fixed a bug in restoring a workspace, which prevented GAP from
+    saving the history if a workspace was loaded during startup
+    ([#2578](https://github.com/gap-system/gap/pull/2578)).
+
+### New and updated packages since GAP 4.9.1
+
+This release contains updated versions of 22 packages from GAP 4.9.1
+distribution. Additionally, it has three new packages. The new
+**JupyterKernel** package by Markus Pfeiffer provides a so-called
+*kernel* for the Jupyter interactive document system
+(<https://jupyter.org/>). This package requires Jupyter to be installed
+on your system (see <https://jupyter.org/install> for instructions). It
+also requires GAP packages **IO**, **ZeroMQInterface**, **json**,
+and also two new packages by Markus Pfeiffer called **crypting** and
+**uuid**, all included into GAP 4.9.2 distribution. The
+**JupyterKernel** package is not yet usable on
+Windows.
+
+
+## GAP 4.9.1 (May 2018)
+
+This is the first public release of GAP 4.9.
+
+### Changes in the core GAP system introduced in GAP 4.9
+
+Major changes:
+
+  - Merged **HPC-GAP** into GAP. For details, please refer to
+    Subsection [3.1-2](chap.md#X7F52B77B7DBACC17) at the end of these
+    release notes.
+
+  - GAP has a new build system, which resolves many quirks and
+    issues with the old system, and will be easier to maintain. For
+    regular users, the usual `./configure && make` should work fine as
+    before. If you are interested in technical details on the new build
+    system, take a look at `README.buildsys.md`.
+
+  - The guidelines for developing GAP packages were revised and
+    moved from the Example package to Reference: Using and Developing
+    GAP Packages ([#484](https://github.com/gap-system/gap/pull/484))
```

Comment:
These refs are broken and will keep breaking as that hash can change. I suggest we just remove this ref, i.e.:
```suggestion
    4.8.5 and breaking `StringFile` ability to work with compressed files.
    \[Reported by Bill Allombert\]
```
well and also get rid of the `\[ ... \]` as mentioned elsewhere.